package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "Simorgh",
   "main": "./src/client.js",
+  "sideEffects": [
+    "./src/app/legacy"
+  ],
```

Comment:
https://webpack.js.org/guides/tree-shaking/#conclusion mentions the sideEffects in the package.json, which gives [exporting hinting to webpack](https://github.com/webpack/webpack/blob/main/examples/side-effects/README.md). So me an Dario looked for the value and it exists in each individual [psammead package](https://github.com/bbc/psammead/blob/2ef5505ed0334857b18f30f531dfbd3df1b46781/packages/components/psammead-brand/package.json#L6), however we didn’t add it as part of the migration. We added this line to the Simorgh root package.json "sideEffects": ["src/app/legacy"] which fixes the psammead-assets problem